src/FSharpVSPowerTools.Core/UntypedAstUtils.fs

```diff
@@ -734,22 +734,22 @@ module Outlining =
             let modstart = mkPos r.StartLine (r.StartColumn+m)
             mkFileIndexRange r.FileIndex modstart r.End
 
-        /// Produce a new range by adding modStart to the StartColumn of `r` 
+        /// Produce a new range by adding modStart to the StartColumn of `r`
         /// and subtracting modEnd from the EndColumn of `r`
         let inline modBoth (r:range) modStart modEnd =
-            let rStart = Range.mkPos r.StartLine (r.StartColumn+modStart) 
-            let rEnd   = Range.mkPos r.EndLine   (r.EndColumn - modEnd) 
+            let rStart = Range.mkPos r.StartLine (r.StartColumn+modStart)
+            let rEnd   = Range.mkPos r.EndLine   (r.EndColumn - modEnd)
             mkFileIndexRange r.FileIndex rStart rEnd
 
-        let inline ofAttributes (attrs:SynAttributes) =            
+        let inline ofAttributes (attrs:SynAttributes) =
             match attrs with | [] -> range () | _  -> startToEnd attrs.[0].Range attrs.[List.length attrs - 1].ArgExpr.Range
 
 
-    ///  Scope indicates the way a range/snapshot should be collapsed. |Scope.Scope.Same| is for a scope inside 
-    ///  some kind of scope delimiter, e.g. `[| ... |]`, `[ ... ]`, `{ ... }`, etc.  |Scope.Below| is for expressions 
-    ///  following a binding or the the right hand side of a pattern, e.g. `let x = ...` 
-    type Collapse = 
-        | Below = 0 
+    ///  Scope indicates the way a range/snapshot should be collapsed. |Scope.Scope.Same| is for a scope inside
+    ///  some kind of scope delimiter, e.g. `[| ... |]`, `[ ... ]`, `{ ... }`, etc.  |Scope.Below| is for expressions
+    ///  following a binding or the the right hand side of a pattern, e.g. `let x = ...`
+    type Collapse =
+        | Below = 0
         | Same = 1
```

Comment:
because it's being added to other value types and they're all ints, @dungpa agreed